client/my-sites/hosting/main.js

```diff
@@ -8,6 +8,12 @@ import { connect } from 'react-redux';
 import { map } from 'lodash';
 import { localize } from 'i18n-calypso';
 
+/**
+ * WordPress dependencies
+ */
+
+import { useState } from '@wordpress/element';
```

Comment:
🤔 Pretty sure we should pull from React here in the Calypso context.

Reply:
Yeh, was getting confused with gutenberg :-)